src/NamespacedPathStackResolverFactory.php

```diff
@@ -6,18 +6,53 @@
 
 use Psr\Container\ContainerInterface;
 
+use function array_filter;
+use function is_array;
+use function is_string;
+use function reset;
+
+/**
+ * @psalm-internal Mezzio\LaminasView
+ * @psalm-internal MezzioTest\LaminasView
+ */
 final class NamespacedPathStackResolverFactory
 {
     public function __invoke(ContainerInterface $container): NamespacedPathStackResolver
     {
+        /** @psalm-var mixed $config */
         $config = $container->has('config') ? $container->get('config') : [];
-        $config = $config['templates'] ?? [];
+        $config = is_array($config) ? $config : [];
```

Comment:
This is just me losing historic context: wasn't this an `ArrayObject`, at some point in history?